src/strategies/squid-dao/schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Strategy",
+  "definitions": {
+    "Strategy": {
+      "title": "Strategy",
+      "type": "object",
+      "properties": {
+        "decimals": {
```

Comment:
Thanks for adding this schema :) 
Can you also add symbol param here, for example: 
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of/schema.json#L9-L13

Reply:
Done!